### DISCUSSION TITLE
"Invalid Access to Memory Location" crash

### DISCUSSION THREAD

**Discussion by ghost** [winsiderss/systeminformer#1943] [Questions / Answers] [answered]
When I navigate to Process>Properties>Memory for Ubisoft Connect (UPC.exe and UplayWebCore.exe), I get "Invalid Access to Memory Location" crash, even if I use Kernel-Mode driver.

**Comment by dmex** [comment, marked as answer]
https://github.com/winsiderss/systeminformer/issues/1983
